pages/soundtrack/soundtrack.pixy

```diff
@@ -37,6 +37,13 @@ component SoundTrackPage(track *arn.SoundTrack, user *arn.User)
 							li
 								a.soundtrack-link(href=link.URL, target="_blank")= link.Title
 
+			if len(relatedTracks) > 0
+				.widget.mountable
+					h3.widget-title Related SoundTracks
```

Comment:
I really like short titles, "Related" alone looks better design-wise and is still understandable from a user perspective.